packages/core/src/modules/credentials/__tests__/CredentialService.test.ts

```diff
@@ -38,6 +38,8 @@ import { CredentialService } from '../services'
 import { CredentialProblemReportMessage } from './../messages/CredentialProblemReportMessage'
 import { credDef, credOffer, credReq } from './fixtures'
 
+import { CredentialMetadataKeys } from '@aries-framework/core'
```

Comment:
Is the package import (instead of relative) on purpose?

```suggestion
import { CredentialMetadataKeys } from '@aries-framework/core'
```